requirements-dev.txt

```diff
@@ -9,3 +9,4 @@ pytest-django==4.8.0
 pytest==8.3.5
 pytest-cov==6.1.1
 coverage==7.7.0
+pillow==11.2.1
```

Comment:
This should already been added in requirements-app.txt so you don't need it here.